stablehlo/reference/Ops.cpp

```diff
@@ -372,6 +465,52 @@ SmallVector<InterpreterValue> eval(Region &region,
       auto operand = scope.findTensor(convertOp.getOperand());
       auto result = evalConvertOp(operand, convertOp.getType());
       scope.add(convertOp.getResult(), result);
+    } else if (auto convolutionOp = dyn_cast<ConvolutionOp>(op)) {
+      auto lhs = scope.findTensor(convolutionOp.getLhs());
+      auto rhs = scope.findTensor(convolutionOp.getRhs());
+      auto rank = lhs.getRank();
+
+      SmallVector<int64_t> windowStrides(rank - 2, 1);
+      if (auto windowStridesAttr = convolutionOp.getWindowStridesAttr())
+        windowStrides = SmallVector<int64_t>(windowStridesAttr.asArrayRef());
```

Comment:
Nit: I think `auto windowStrides = convolutionOp.getWindowStrides().value_or(SmallVector<int64_t, 0>{})` should work.

(Actually it's unclear to me that `SmallVector` is needed here, you could do something similar with `ArrayRef`.)

Reply:
We want default value to be filled with ones. `value_or` actually returns an `ArrayRef`, so there would be a type mismatch (and creating defaults with `ArrayRef` doesn't offer more readability compared to using `SmallVector` for the type).